app/controllers/api/v3/organizations.rb

```diff
@@ -44,14 +44,14 @@ def allowed_write_organizations
 
           permitted = declared(params, include_missing: false)
           organization = Organization.new(
-            permitted.to_hash.except("locations").merge(auto_user_id: current_user.id)
+            permitted.to_h.except("locations").merge(auto_user_id: current_user.id)
```

Comment:
Calling `to_hash` on `params` is deprecated in Rails because params no longer inherit from `Hash`. `to_h` is defined, however.

Reply:
I believe the root cause of this is that the services are tested using hashes rather than permitted parameters (which I think is reasonable, maybe even a good thing).

Something cludgy that might work is switching the `.to_hash` to `.as_json` - but, this shouldn't be resolved before shipping this PR. We can discuss.